src/Symfony/Bundle/FrameworkBundle/Routing/AttributedRouteControllerLoader.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Routing;
+
+/**
+ * AttributedRouteControllerLoader is an implementation of AttributeClassLoader
+ * that sets the '_controller' default based on the class and method names.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ * @author Alexandre Daubois <alex.daubois@gmail.com>
+ */
+class AttributedRouteControllerLoader extends AnnotatedRouteControllerLoader
```

Comment:
```suggestion
class AttributeRouteControllerLoader extends AnnotatedRouteControllerLoader
```
as `attributed` has a different meaning

Reply:
Updated, thanks!